### PR TITLE
work

### PR DESCRIPTION
**Pull request by Eduardo-Lash**
lets see if this works

### CODE REVIEW

**Review by cyrillegin — CHANGES_REQUESTED** [review]
Almost all of the changes here are not your code. For this PR to be approved, you need to only add your own code. The changes that make it look like you're committing this seems to be from you moving files around? Reorganizing is fine but make sure that the commit is using a move rather than a delete/add.

File: Pico map/Assets/Scripts/GestureCreator.cs [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        y = y *  sceneHeight / imageCapture.cameraHeight;
+        Vector3 final = new Vector3 (x, y, 400);
+        print(x + " " + y);
+        return final;
+    }
+
+    Vector3 GetMidpoint(Vector3 start, Vector3 end) {
+        return new Vector3((start.x + end.x) / 2, (start.y + end.y) / 2, 400);
+    }
+}
```
Comment (cyrillegin) [inline Pico map/Assets/Scripts/GestureCreator.cs:113]: None of this is your code.

File: Pico map/Assets/Scripts/ImageCapture.cs [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    //             float newBlue = (startData[i * j].b + startData[i * j + 1].b + startData[i * j - 1].b + startData[(i - 1) * j].b + startData[(i + 1) * j].b) / 5;
+    //
+    //             resultColor[counter] = new Color32((byte) newRed, (byte) newGreen, (byte) newBlue, 1);
+    //             counter++;
+    //         }
+    //     }
+    //     return resultColor;
+    // }
+
+}
```
Comment (cyrillegin) [inline Pico map/Assets/Scripts/ImageCapture.cs:106]: None of this is your code.

File: Pico map/Assets/Scripts/webCam.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+	// Use this for initialization
+	void Start () {
+		
+	}
+	
+	// Update is called once per frame
+	void Update () {
+		
+	}
+}
```
Comment (cyrillegin) [inline Pico map/Assets/Scripts/webCam.cs:16]: Does this do anything?

File: Pico map/Assets/Scenes/Map.unity [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+      propertyPath: m_LocalRotation.w
+      value: 0.7071068
+      objectReference: {fileID: 0}
+    - target: {fileID: 4931552512806170, guid: c9ca8cdd96bdb9e4999db2bcd856c49b, type: 2}
+      propertyPath: m_RootOrder
+      value: 0
+      objectReference: {fileID: 0}
+    m_RemovedComponents: []
+  m_ParentPrefab: {fileID: 100100000, guid: c9ca8cdd96bdb9e4999db2bcd856c49b, type: 2}
+  m_IsPrefabParent: 0
```
Comment (cyrillegin) [inline Pico map/Assets/Scenes/Map.unity:152]: What changes to the scene did you make? You need to add details in your commit messages

File: Pico map/Assets/Scripts/quit.cs [hunk 5]
```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class quit : MonoBehaviour
```
Comment (cyrillegin) [inline Pico map/Assets/Scripts/quit.cs:5]: Class names should be capitalized

File: Pico map/Assets/Scripts/quit.cs [hunk 6]
```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
```
Comment (cyrillegin) [inline Pico map/Assets/Scripts/quit.cs:4]: What does this script do? Every script should have some header comments telling us what it is doing.

File: Pico map/Assets/Scripts/webCam.cs [hunk 7]
```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
```
Comment (cyrillegin) [inline Pico map/Assets/Scripts/webCam.cs:4]: What does this script do? Every script should have some header comments telling us what it is doing.

File: Pico map/Assets/Scripts/webCam.cs [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+public class webCam : MonoBehaviour {
```
Comment (cyrillegin) [inline Pico map/Assets/Scripts/webCam.cs:5]: Capital classes.

File: Pico map/Assets/Scripts/quit.cs (hunk 6 above)
Comment (Eduardo-Lash) [reply Pico map/Assets/Scripts/quit.cs:4]: yes this dose it exits the build

File: Pico map/Assets/Scripts/webCam.cs (hunk 7 above)
Comment (Eduardo-Lash) [reply Pico map/Assets/Scripts/webCam.cs:4]: this one you can throw it dose nothing

File: Pico map/Assets/Scripts/webCam.cs (hunk 8 above)
Comment (Eduardo-Lash) [reply Pico map/Assets/Scripts/webCam.cs:5]: thats a pref.

File: Pico map/Assets/Scripts/quit.cs (hunk 5 above)
Comment (Eduardo-Lash) [reply Pico map/Assets/Scripts/quit.cs:5]: thats a pref.

File: Pico map/Assets/Scripts/GestureCreator.cs (hunk 1 above)
Comment (Eduardo-Lash) [reply Pico map/Assets/Scripts/GestureCreator.cs:113]: yah its yours

File: Pico map/Assets/Scripts/ImageCapture.cs (hunk 2 above)
Comment (Eduardo-Lash) [reply Pico map/Assets/Scripts/ImageCapture.cs:106]: yah its yours

File: Pico map/Assets/Scripts/GestureCreator.cs (hunk 1 above)
Comment (cyrillegin) [reply Pico map/Assets/Scripts/GestureCreator.cs:113]: So remove it.

File: Pico map/Assets/Scripts/ImageCapture.cs (hunk 2 above)
Comment (cyrillegin) [reply Pico map/Assets/Scripts/ImageCapture.cs:106]: So remove it.

File: Pico map/Assets/Scripts/quit.cs (hunk 5 above)
Comment (cyrillegin) [reply Pico map/Assets/Scripts/quit.cs:5]: You're right, so change it.

File: Pico map/Assets/Scripts/quit.cs (hunk 6 above)
Comment (cyrillegin) [reply Pico map/Assets/Scripts/quit.cs:4]: I'm not specifically asking what this script does, just that every script needs comments detailing what it does.

File: Pico map/Assets/Scripts/webCam.cs (hunk 8 above)
Comment (cyrillegin) [reply Pico map/Assets/Scripts/webCam.cs:5]: You're right, so change it.

File: Pico map/Assets/Scripts/webCam.cs (hunk 7 above)
Comment (cyrillegin) [reply Pico map/Assets/Scripts/webCam.cs:4]: I'm not specifically asking what this script does, just that every script needs comments detailing what it does.

**Comment by RandallWoodall** [comment]
Branch has serious issues that need to be resolved before being merged into master.

**Pull request closed without merging**